### PR TITLE
Add recipe for harmony

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds the recipe for the R harmony package by @ilyakorsunsky - see https://github.com/immunogenomics/harmony.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by pinin4fjords** [comment]
@BiocondaBot please add label

File: recipes/r-harmony/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+test:
+  commands:
+    - $R -e "library('harmony')"
+
+about:
+  home: https://github.com/immunogenomics/harmony
+  dev_url: https://github.com/immunogenomics/harmony 
+  license: GPL-3
+  summary: Fast, sensitive and accurate integration of single-cell data with Harmony
+  license_family: GPL
```
Comment (mbargull) [inline recipes/r-harmony/meta.yaml:56]:
```suggestion
  license: GPL-3.0-only
  license_file: LICENSE
  summary: Fast, sensitive and accurate integration of single-cell data with Harmony
```

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

File: recipes/r-harmony/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  name: r-harmony
+  version: {{ version }}
+
+source:
+    url: https://github.com/immunogenomics/harmony/archive/{{ version }}.tar.gz
+    sha256: 1243bec2c9cd96d32ee48cafc74795bf18c3b50789e388ef8d80e4440940ae2b 
+
+build:
+  number: 0
+  noarch: generic
```
Comment (mbargull) [inline recipes/r-harmony/meta.yaml:13]:
```suggestion
```

File: recipes/r-harmony/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
```
Comment (mbargull) [inline recipes/r-harmony/meta.yaml:19]:
```suggestion
  build:
    - {{ compiler('cxx') }}
  host:
```

File: recipes/r-harmony/meta.yaml (hunk 2 above)
Comment (pinin4fjords) [reply recipes/r-harmony/meta.yaml:13]: Ahh, my bad- thanks for the catch!

**Comment by pinin4fjords** [comment]
Thanks very much for helpful review @mbargull

**Comment by mbargull** [comment]
@BiocondaBots, please fetch artifacts

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

**Comment by mbargull** [comment]
@BiocondaBot, please merge

**Pull request merged**